src/form/birth/certificate-handlebars.ts

```diff
@@ -17,38 +17,21 @@ export const certificateHandlebars = {
   certificateDate: 'certificateDate',
   registrar: 'registrar',
   registrationAgent: 'registrationAgent',
-  registrarName: 'registrarName', // @deprecated use registrar.name instead
-  role: 'role', // @deprecated use registrar.role instead
-  registrarSignature: 'registrarSignature', // @deprecated use registrar.signature instead
+  registrarName: 'registrarName',
```

Comment:
Do the deprecated messages have relevance to us? Should they stay?

Reply:
I dont think that makes sense. we can get rid of it